packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -34,7 +34,7 @@ import {
   getCurrentRange,
   isMultiBlockRange,
   getModelsByRange,
-} from '../../__internal__'
+} from '../../__internal__';
```

Comment:
Just saying that we have `pnpm format` for prettifying.

Reply:
I would like to use git hooks to run it automatically before submitting